### ISSUE TITLE
Syntax Error loading add-on in Godot 3.2.3

### ISSUE THREAD

**Issue by CarterG81** [paulloz/godot-ink#26]
Installed the Add-On and immediately got an error.

> Unable to load addon script from path: 'res://addons/paulloz.ink/PaullozDotInk.cs' There seems to be an error in the code, please check the syntax.

**Comment by paulloz** [comment]
Hi, it looks like the error you'd have if you haven't setup the ink dll.  
Did you check the [How to use](https://github.com/paulloz/godot-ink#how-to-use) and [Troubleshooting](https://github.com/paulloz/godot-ink#troubleshooting) sections in the README?

**Comment by CarterG81** [comment]
Yes, and I figured that might be the case. I added the .dll to the root of my Godot project, but the error remained.

Looking at Troubleshooting helped. I'm pretty sure this is a problem on my system with something Microsoft. I've been working on it for a bit, and am pretty certain it has nothing to do with this. I'm doing a repair of VS build tools right now.

Sorry to trouble you. I'll work on it until I get it to work.
